sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/spanner/SpannerIO.java

```diff
@@ -991,6 +1001,24 @@ public WriteGrouped(Write spec) {
       this.spec = spec;
     }
 
+    @Override
+    public void populateDisplayData(DisplayData.Builder builder) {
+      super.populateDisplayData(builder);
+      spec.getSpannerConfig().populateDisplayData(builder);
+      builder.add(
+          DisplayData.item("batchSizeBytes", spec.getBatchSizeBytes())
+              .withLabel("Max batch size in sytes"));
+      builder.add(
+          DisplayData.item("maxNumMutations", spec.getMaxNumMutations())
+              .withLabel("Max number of mutated cells in each batch"));
+      builder.add(
+          DisplayData.item("maxNumRows", spec.getMaxNumRows())
+              .withLabel("Max number of rows in each batch"));
+      builder.add(
+          DisplayData.item("groupingFactor", spec.getGroupingFactor())
+              .withLabel("Number of batches to sort over"));
+    }
+
```

Comment:
I added some commits to tweak these descriptions a bit. @nielm can you confirm that they're still correct?

Also I wonder if you can re-use the implementation in `Write` by calling `spec.populateDisplayData(builder)` here instead?

Reply:
LGTM. 
I extracted a single method in Write to populate the displayData with parameters to avoid repeating code.